app/models/rental.py

```diff
@@ -1,4 +1,36 @@
 from app import db
+from app.models.customer import Customer
+from app.models.video import Video
 
 class Rental(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    customer_id = db.Column(db.Integer, db.ForeignKey('customer.id'))
+    video_id = db.Column(db.Integer, db.ForeignKey('video.id'))
+    due_date = db.Column(db.DateTime)
+    checked_in = db.Column(db.Boolean, default=False)
+
+    def rental_to_dict(self):
+        return {
+            "customer_id": self.customer_id,
+            "video_id": self.video_id,
+            "due_date": self.due_date,
+            "videos_checked_out_count": len(self.customer.rentals),
+            "available_inventory": self.video.total_inventory - len(self.video.rentals),
```

Comment:
Great work using the rentals column to calculate these values. 

You might consider making helper methods on the `video` and `customer` models that return these values such as 

```
#customer.py

def get_videos_checked_out_count(self):
    return len(self.rentals)
```